operators/multiclusterobservability/manifests/base/config/metrics_allowlist.yaml

```diff
@@ -137,6 +137,7 @@ data:
       - __name__="container_memory_rss",container!=""
       - __name__="container_memory_swap",container!=""
       - __name__="container_memory_working_set_bytes",container!=""
+      - product="acm"
```

Comment:
I do not think we want to collect thanos metrics and put them into thanos again. It may trigger the unnecessary update. Just put the data into openshift monitoring is fine for me. if you agree with that, we can remove `product="acm"` label in original metrics.

Reply:
ok, udpated